zanata-war/src/main/java/org/zanata/dao/LocaleMemberDAO.java

```diff
@@ -56,6 +56,24 @@ public LocaleMemberDAO(Session session) {
         return query.list();
     }
 
+    public List<HLocaleMember> findActiveMembers(LocaleId localeId) {
+        StringBuilder sb = new StringBuilder();
+        sb.append("from HLocaleMember ")
+            .append("where id.supportedLanguage.localeId = :localeId ")
+            .append("and (isTranslator = :isTranslator ")
+            .append("or isReviewer = :isReviewer ")
+            .append("or isCoordinator = :isCoordinator) ")
+            .append("order by lower(id.person.name)");
+        Query query = getSession().createQuery(sb.toString());
+        query.setParameter("localeId", localeId);
+        query.setParameter("isTranslator", true);
+        query.setParameter("isReviewer", true);
+        query.setParameter("isCoordinator", true);
```

Comment:
There's no need to declare these as constants. You could just modify the query to be:

```java
.append("and (isTranslator = true ")
.append("or isReviewer = true ")
.append("or isCoordinator = true) ")
```
(or something similar) This would save a tiny bit of overhead when running the query (and perhaps when caching it).